x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -4349,8 +4344,6 @@
     "xpack.apm.serviceMap.focusMapButtonText": "聚焦地图",
     "xpack.apm.serviceMap.invalidLicenseMessage": "要访问服务地图，必须订阅 Elastic 白金级许可证。使用该许可证，您将能够可视化整个应用程序堆栈以及 APM 数据。",
     "xpack.apm.serviceMap.serviceDetailsButtonText": "服务详情",
-    "xpack.apm.serviceMap.subtypePopoverMetric": "子类型",
-    "xpack.apm.serviceMap.typePopoverMetric": "类型",
```

Comment:
These translations didn't chance, just the Id, i think we can safely update just the ids here.

Reply:
I'm going to leave them but I'll let the i18n team know that they should be the same.